graphql/schema.graphql

```diff
@@ -9,6 +9,7 @@ type Query {
   projects: [GroupedProjects]!
   viewableProjectRefs: [GroupedProjects]!
   githubProjectConflicts(projectId: String!): GithubProjectConflicts!
+  canCreateProject: Boolean!
```

Comment:
Would this make sense under the user object? 
Since this determines if a user can create a project it would make it a little more flexible since we can query against any user instead of only your own user.

Reply:
To further elaborate: 

I think there would be benefit to setting up a "permissions" resolver that returns what permissions a user has, like canAnnotate, CanCreateProject, etc, and add to that as we keep needing more and more permissions for users. 

However, I think that if we want to set that up, we should keep this as is, and then create a ticket for that refactor since this is needed now, and that refactor would include also pulling out the permissions we have now in other resolvers (I can think of at least one).

Reply:
A permissions resolver makes sense to me. I still think it should belong to a user object since permissions are something a user has. It also helps create a more complete graph and would allow for more flexibility in the future. (e.g) looking up what permissions a user has.

Reply:
Updated to have a permissions resolver! I'm not sure if having userId be part of the permissions actually makes sense but I'm not sure how else to support having userId be passed in.